### ISSUE TITLE
Error looking at a record that has been deleted

### ISSUE THREAD

**Issue by brunochimenes** [Nozbe/WatermelonDB#1331]
I have a product and brand table and on my mobile app, I'm using watermelondb with mysql. On my API, I'm using paranoid, so, I ain't deleting my records from database, but adding a value to deleted_at. So, when I delete a brand on my site , I send an API call and set the deleted_at to that specific brand. But on mobile, the brand record is deleted, but the brand_id still has the value that references the brand that got deleted. How can I handle an ```observe( )``` on my app, since it breaks the application when it tries do find the brand_id that doesn't exists anymore?

![Captura de tela 2022-05-20 165642](https://user-images.githubusercontent.com/88602547/170098581-f425c8d4-50ae-4e2f-8be5-98cf5cfa69d4.png)

I'm using this version:
```
"@nozbe/watermelondb": "^0.23.0",
"@nozbe/with-observables": "^1.4.0",
"react-native": "0.64.1",
```

**Comment by 1mike12** [comment]
are you not "cascading" the deletion on your backend to the product table by setting deleted_at on all products associated with that brand?

**Comment by brunochimenes** [comment]
No, here we are not cascading the registers associated to brand. Do you have any idea what can I do or you need more informations ?

**Comment by KrisLau** [comment]
@brunochimenes You can do this: https://nozbe.github.io/WatermelonDB/Writers.html?highlight=delete#delete-action

**Comment by 3210jr** [comment]
> @brunochimenes You can do this: https://nozbe.github.io/WatermelonDB/Writers.html?highlight=delete#delete-action

new link for future wanderers: https://watermelondb.dev/docs/Writers#delete-action
